front/app/containers/Admin/pagesAndMenu/containers/CustomPages/messages.ts

```diff
@@ -0,0 +1,58 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  newCustomPageMetaTitle: {
+    id: 'app.containers.Admin.pagesAndMenu.containers.CustomPages.newCustomPageMetaTitle',
+    defaultMessage: 'Meta title',
```

Comment:
I see how the title can be valuable (in the opposite of the description), but what does "Meta title" mean? Shouldn't it be sth like "New custom page"?